pepper-apis/src/main/java/org/broadinstitute/ddp/client/GoogleBucketClient.java

```diff
@@ -0,0 +1,28 @@
+package org.broadinstitute.ddp.client;
+
+import com.google.auth.Credentials;
+import com.google.cloud.storage.Bucket;
+import com.google.cloud.storage.StorageOptions;
+
+/**
+ * A client wrapper around Google Bucket services.
+ */
+public class GoogleBucketClient {
```

Comment:
Starting to consolidate Google Bucket usages into this class. Will do more refactors separately.